.ci/setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+test_repo="github.com/clearcontainers/tests"
+
+# Clone Tests repository.
+go get "$test_repo"
+
+test_repo_dir="${GOPATH}/src/${test_repo}"
+
+# Check the commits in the branch
```

Comment:
If we are having the same `checkcommits` config across all the repos in the project, do you think we can re-factor this call into a single script that lives in the https://github.com/kata-containers/tests repo.
Even if for now that script then grabs `checkcommits` itself from https://github.com/clearcontainers/tests whilst we work out if we import that wholesale or not into our kata tests repo?

> note - this stems from the long standing idea that we needed to factorise the go static checks out of their individual repos and into a single file under Clear Containers - something we never quite got around to doing.
> let's not end up in that place again :-)

Reply:
I guess to be even more explicit. If this script did something like:
```
test_repo="github.com/clearcontainers/tests"
# Clone Tests repository.
go get "$test_repo"
# Run the 'checkers'
${tests_repo}/.ci/run_all_static_checks.sh
```

and then we can formalise and keep all the static checking (checkmetrics, static go linters etc.) in one place.

Obviously, this same comment goes for all the other repos under kata where you have the same PRs sat :-)

Reply:
I like the idea. Respinning bytes... :)

Reply:
Hi @grahamwhaley - how about https://github.com/kata-containers/tests/pull/7?